app/lib/submission_builder/ty2024/states/id/id_return_xml.rb

```diff
@@ -52,8 +52,6 @@ def supported_documents
               }
             end
 
-            supported_docs += form1099rs
-            supported_docs += form1099gs
```

Comment:
noticed while looking at id xml output that we were seeing errors for `form1099r`
<img width="1892" alt="Screenshot 2024-10-30 at 2 24 47 PM" src="https://github.com/user-attachments/assets/88d77ea8-43e4-4a66-be25-85567ac9f503">

Turns out ID / NC does not want income document xml, so removing them here